packages/flutter/lib/src/material/dropdown_menu.dart

```diff
@@ -376,6 +383,40 @@ class DropdownMenu<T> extends StatefulWidget {
   /// Defaults to null.
   final EdgeInsets? expandedInsets;
 
+  /// When [DropdownMenu.enableFilter] is true, this callback is used to
+  /// compute the list of filtered items.
+  ///
+  /// {@tool snippet}
+  ///
+  /// In this example the `filterCallback` returns the items that contains the
+  /// trimmed query.
+  ///
+  /// ```dart
+  /// DropdownMenu<Text>(
+  ///   filterCallback: (List<DropdownMenuEntry<Text>> entries, String filter) {
```

Comment:
Since enableFilter is false by default, shouldn't this filterCallback not do anything?

Reply:
Looks like this is fixed now :+1: